subsys/bluetooth/mesh/settings.c

```diff
@@ -104,10 +104,6 @@ static int mesh_commit(void)
 		return 0;
 	}
 
-	if (IS_ENABLED(CONFIG_BT_MESH_PB_GATT)) {
```

Comment:
Why did you remove this? Could you elaborate a bit?

Reply:
I added this explanation in the commit message (Realized I'm missing a "call", will fix):
> The [call] to `bt_mesh_pb_gatt_srv_disable` in settings.c is removed. This is
called in `mesh_commit`, and is not needed as this is only used during
boot, at which point the Mesh provisioning service has not yet been
enabled/registered, meaning the function does nothing.

Is this sufficient?